polaris-core/src/main/java/org/apache/polaris/core/auth/PolarisAuthorizer.java

```diff
@@ -41,4 +41,18 @@ void authorizeOrThrow(
       @Nonnull PolarisAuthorizableOperation authzOp,
       @Nullable List<PolarisResolvedPathWrapper> targets,
       @Nullable List<PolarisResolvedPathWrapper> secondaries);
+
+  void authorizeOrThrow(
```

Comment:
Would it be possible to add `default` (possibly inefficient) implementations to the new methods to simplify transition in downstream projects? This is just a matter of convenience, so optional.